app/src/ui/lib/configure-git-user.tsx

```diff
@@ -131,16 +131,15 @@ export class ConfigureGitUser extends React.Component<
     )
   }
 
-  private onNameChange = (event: React.FormEvent<HTMLInputElement>) => {
+  private onNameChange = (name: string) => {
     this.setState({
-      name: event.currentTarget.value,
+      name,
       email: this.state.email,
```

Comment:
I think we can drop email and avatarUrl here and just update name. I realize that this is ancient code but it's a nice cleanup.